vocab/types.json

```diff
@@ -1148,15 +1148,15 @@
         "translatableTo": null
     },
     "https://openminds.ebrains.eu/core/GRIDID": {
-        "description": null,
+        "description": "A GRID (Global Research Identifier Database) identifier.",
         "label": "GRIDID",
         "schemas": [
             "core/v3/miscellaneous/GRIDID",
             "core/v4/digitalIdentifier/GRIDID"
         ]
     },
     "https://openminds.ebrains.eu/core/HANDLE": {
-        "description": null,
+        "description": "A persistent identifier for an information resource provided by the Handle System of the Corporation for National Research Initiatives.",
```

Comment:
I suggest to follow the same logic as in "Structured information about a digital object identifier, as standardized by the International Organization for Standardization."